### PR TITLE
This is my first pull request

### PR DESCRIPTION
**Pull request by James-Mulcahy**
closes #  <--- some number from earlier issues #?

### CODE REVIEW

**Comment by Jin2017GitHub01** [comment]
What's up!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # GitHub for Developers
+#
```
Comment (Jin2017GitHub01) [inline README.md:3]: gOOD

Comment (James-Mulcahy) [reply README.md:3]: :+1:

**Review by Jin2017GitHub01 — CHANGES_REQUESTED** [review]
This looks dangerous.

**Review by Jin2017GitHub01 — APPROVED** [review]
Bomb approved

**Pull request merged**